en/docs/learn/examples/template-examples/using-endpoint-templates.md

```diff
@@ -95,25 +95,15 @@ In this example, the endpoint template is configured to invoke the endpoints bas
        <resource methods="GET" uri-template="/foo">
           <inSequence>
              <call>
-                <endpoint template="ep_template"
-                          uri="http://run.mocky.io/v3/8ca6aa42-ee0a-47a8-8007-e93abbb95b87">
-                   <parameter name="codes" value="10001,10002"/>
-                   <parameter name="retries" value="2"/>
-                   <parameter name="factor" value="1.0"/>
-                </endpoint>
+                <endpoint key="ep1"/>
```

Comment:
Let;s keep the original values. As this shows from the API it calls the endpoint template.